exp/controllers/machinepool_controller_test.go

```diff
@@ -29,10 +29,10 @@ import (
 	"k8s.io/utils/pointer"
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
 	expv1 "sigs.k8s.io/cluster-api/exp/api/v1alpha3"
+	"sigs.k8s.io/cluster-api/test/helpers"
```

Comment:
Changed this to use the `helpers` package, since patching the conditions requires the ResourceVersion to be set.